### ISSUE TITLE
Accessibility Inspector (enabled by KIF) occludes labels preventing them being tapped by UI tests

### ISSUE THREAD

**Issue by ratkins** [kif-framework/KIF#762]
Is there any way to force KIF to start the Accessibility Inspector in minimized mode, or turn it off altogether when KIF is done with it?

It occludes labels in our interface that subsequent Xcode 7 UI tests try, and then fail, to successfully tap.

**Comment by phatmann** [comment]
@TBonnin, @joemasilotti: any ideas?

**Comment by ratkins** [comment]
Having had a closer look, it appears that KIF *tries* to do this (on `KIFAccessibilityEnabler.m/_resetAccessibilityInspector` which should be called via `XCTestObservation`'s `– testBundleDidFinish:`?)

It seems to work with the latest version of kif when I run the tests from Xcode in a freshly-reset simulator in a normal user account. But I'm getting my [test failures](http://stackoverflow.com/questions/34451515/how-do-i-turn-off-the-accessibility-inspector-in-the-ios-9-simulator/34500882#34500882) on the build server, and I can't se how to reset the simulators that the secret Xcode Bot runs in its own user account.

**Comment by phatmann** [comment]
Xcode Bot uses the installed version of Xcode and its simulators to run tests. So just open up the Simulator on the server machine and reset it.

**Comment by ratkins** [comment]
Is the configuration different for a simulator run from the Bot and a simulator run from a user account though?

**Comment by ratkins** [comment]
Aha, got it: had to run this as a "Before Integration" trigger on the bot:
````
/usr/bin/osascript -e 'tell application "iOS Simulator" to quit'
/usr/bin/osascript -e 'tell application "Simulator" to quit'
/usr/bin/xcrun simctl erase all
````

**Comment by jznadams** [comment]
I just ran into the same problem running the tests via xctool with Jenkins. In my case, running `xcrun simctil erase all` did not seem to help. The next time the simulator runs, the accessibility inspector is positioned directly over the left navigation bar button which breaks many of my tests. Manually dragging it out of the way (over the device status bar at the very top in the middle) seemed to work for subsequent tests. Perhaps it would be worth adding a mention of this to the troubleshooting section of the readme? I think that would've saved me some time trying to figure out what was wrong with my tests.
